src/main/apiClient.ts

```diff
@@ -15,8 +15,14 @@ import { popupSize } from './providerPopupSize'
 import { createHttpClient, HttpClient } from './httpClient'
 import { computePkceParams, PkceParams } from './pkceService'
 
-export type SignupParams = { data: SignupProfile; saveCredentials?: boolean; auth?: AuthOptions; redirectUrl?: string }
-export type UpdateEmailParams = { accessToken: string; email: string; redirectUrl?: string }
+export type SignupParams = {
+  data: SignupProfile
+  returnToAfterConfirmEmail?: string
```

Comment:
We do not standardise the parameter name for all the use cases ?

Reply:
non

Reply:
@gpartenet @jcbee @gbersac I think in terms of english, the terms should be:
`returnToAfterConfirmEmail` -> `returnToAfterEmailConfirmation`
and for password:
`returnToAfterResetPassword ` -> `returnToAfterPasswordReset`

Reply:
Lastly, could we place both redirect URL parameters alongside each other?

Reply:
I changed to those params.